### PR TITLE
Fix several recently added StyleCop tuple rules

### PR DESCRIPTION
**Pull request by stephentoub**

### CODE REVIEW

File: src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs [hunk 1]
```diff
@@ -329,11 +329,11 @@ private static unsafe string ToUpperHexString(ReadOnlySpan<byte> ba)
         {
             fixed (byte* baPtr = ba)
             {
-                return string.Create(ba.Length * 2, (new IntPtr(baPtr), ba.Length), (span, args) =>
+                return string.Create(ba.Length * 2, (Ptr: new IntPtr(baPtr), ba.Length), (span, args) =>
```
Comment (danmosemsft) [inline src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs]: Why is Ptr: needed on the first  parameter but no name required for the second one?

**Review by danmoseley — APPROVED** [review]

File: src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs (hunk 1 above)
Comment (stephentoub) [reply src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs]: The compiler infers the name of the second one based on the accessed property "Length".  It has nothing to infer on the first one.

Comment (danmosemsft) [reply src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs]: Ah, clever.

**Comment by jashook** [comment]
@stephentoub this change seems to have broken coreclr's build

**Comment by jashook** [comment]
You can see our commits, https://github.com/dotnet/coreclr/commit/a43b2abba24c90058b1eeaf12d2241a66619acde is the first failing build.

**Comment by stephentoub** [comment]
Two commits crossed.  Will fix.

**Pull request merged**